layers/vulkan/generated/state_tracker_helper.cpp

```diff
@@ -0,0 +1,1710 @@
+        features->shaderSampledImageArrayDynamicIndexing = core_features->shaderSampledImageArrayDynamicIndexing == VK_TRUE;
+        features->shaderStorageBufferArrayDynamicIndexing = core_features->shaderStorageBufferArrayDynamicIndexing == VK_TRUE;
+        features->shaderStorageImageArrayDynamicIndexing = core_features->shaderStorageImageArrayDynamicIndexing == VK_TRUE;
+        features->shaderClipDistance = core_features->shaderClipDistance == VK_TRUE;
+        features->shaderCullDistance = core_features->shaderCullDistance == VK_TRUE;
+        features->shaderFloat64 = core_features->shaderFloat64 == VK_TRUE;
+        features->shaderInt64 = core_features->shaderInt64 == VK_TRUE;
+        features->shaderInt16 = core_features->shaderInt16 == VK_TRUE;
+        features->shaderResourceResidency = core_features->shaderResourceResidency == VK_TRUE;
+        features->shaderResourceMinLod = core_features->shaderResourceMinLod == VK_TRUE;
+        features->sparseBinding = core_features->sparseBinding == VK_TRUE;
+        features->sparseResidencyBuffer = core_features->sparseResidencyBuffer == VK_TRUE;
+        features->sparseResidencyImage2D = core_features->sparseResidencyImage2D == VK_TRUE;
+        features->sparseResidencyImage3D = core_features->sparseResidencyImage3D == VK_TRUE;
+        features->sparseResidency2Samples = core_features->sparseResidency2Samples == VK_TRUE;
+        features->sparseResidency4Samples = core_features->sparseResidency4Samples == VK_TRUE;
+        features->sparseResidency8Samples = core_features->sparseResidency8Samples == VK_TRUE;
+        features->sparseResidency16Samples = core_features->sparseResidency16Samples == VK_TRUE;
+        features->sparseResidencyAliased = core_features->sparseResidencyAliased == VK_TRUE;
+        features->variableMultisampleRate = core_features->variableMultisampleRate == VK_TRUE;
+        features->inheritedQueries = core_features->inheritedQueries == VK_TRUE;
+    }
+
+    for (const VkBaseInStructure *pNext = reinterpret_cast<const VkBaseInStructure *>(pCreateInfo->pNext); pNext != nullptr;
+         pNext = pNext->pNext) {
+        switch (pNext->sType) {
+            case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_16BIT_STORAGE_FEATURES: {
+                const VkPhysicalDevice16BitStorageFeatures *enabled =
+                    reinterpret_cast<const VkPhysicalDevice16BitStorageFeatures *>(pNext);
+                features->storageBuffer16BitAccess =
+                    features->storageBuffer16BitAccess || enabled->storageBuffer16BitAccess == VK_TRUE;
+                features->uniformAndStorageBuffer16BitAccess =
+                    features->uniformAndStorageBuffer16BitAccess || enabled->uniformAndStorageBuffer16BitAccess == VK_TRUE;
+                features->storagePushConstant16 = features->storagePushConstant16 || enabled->storagePushConstant16 == VK_TRUE;
```

Comment:
not a deal breaker, but would nice to generate this as 

```suggestion
                features->storagePushConstant16 |= enabled->storagePushConstant16 == VK_TRUE;
```

Reply:
It's technically not the same thing (logical vs bitwas or), which is why I avoided it, but I see VVL does this all over so fine, done.